bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/intellichlor.xml

```diff
@@ -95,13 +111,21 @@
 		<item-type>Switch</item-type>
 		<label>Low Voltage</label>
 		<description>Chlorinator cell is at a low voltage.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Voltage</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="lowWaterTemp">
 		<item-type>Switch</item-type>
 		<label>Low Water Temperature</label>
 		<description>Water temperature is too low for chlorine generation.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Temperature</tag>
```

Comment:
This is a switch, so not really a temperature reading.  Maybe "Water" is better?

Reply:
Is the switch read only? If so it is a Status. And does the status relate to temperature? If so it is indeed Status.Temperature..